app-template/tests/integration.rs

```diff
@@ -20,16 +19,19 @@ use cosmwasm_std::{coins, Addr};
 #[allow(clippy::type_complexity)]
 fn setup(
     count: i32,
-) -> anyhow::Result<(AbstractClient<Mock>, Application<Mock, AppInterface<Mock>>)> {
+) -> anyhow::Result<(
+    AbstractClient<MockBech32>,
+    Application<MockBech32, AppInterface<MockBech32>>,
+)> {
     // Create a sender and mock env
-    let mock = Mock::new("sender");
+    let mock = MockBech32::new("mock");
     let sender = mock.sender();
     let namespace = Namespace::from_id(APP_ID)?;
 
     // You can set up Abstract with a builder.
     let client = AbstractClient::builder(mock).build()?;
     // The client supports setting balances for addresses and configuring ANS.
-    client.set_balance(&sender, &coins(123, "ucosm"))?;
+    client.set_balance(sender, &coins(123, "ucosm"))?;
```

Comment:
Clippy is mad if last usage of `sender` non-consuming now https://rust-lang.github.io/rust-clippy/master/index.html#/needless_borrows_for_generic_args